src/Framework/TestCase.php

```diff
@@ -70,6 +66,12 @@ public function run($method = NULL)
 	 */
 	public function runTest($method, array $args = NULL)
 	{
+		if (!method_exists($this, $method)) {
+			throw new TestCaseException("Method '$method' does not exist.");
```

Comment:
Actually this is not needed. The `ReflectionException` will be thrown. But this have a straightforward message.